projects/email_alert_notifications/email_alert_notifications.tf

```diff
@@ -0,0 +1,55 @@
+provider "aws" {}
```

Comment:
This isn't required because [it is supplied as part of common variables](https://github.com/alphagov/govuk-terraform-provisioning/blob/master/configs/common_variables.tf)